defaults/main.yml

```diff
@@ -57,7 +58,13 @@ openio_oioswift_filter_proxy_logging:
 
 openio_oioswift_filter_cache:
   use: "egg:swift#memcache"
-  memcache_servers: "{{ openio_oioswift_bind_address }}:6019"
+  memcache_servers: "{% if openio_oioswift_inventory_groupname %}\
+    {{ groups[openio_oioswift_inventory_groupname] |\
+    map('extract', hostvars, ['ansible_' + openio_oioswift_bind_interface]['ipv4']['address']) | join(':6019,') }}:6019\
```

Comment:
Isn't that adding trailing ":6019" ?

Reply:
you prefer a `for` loop ? with `{% if not loop.last %},{% endif %}{% endfor %}`